### PR TITLE
feat: do not render an icon, if only one level is defined

### PR DESCRIPTION
**Pull request by natterstefan**
resolves https://github.com/editor-js/header/issues/40

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -113,46 +113,50 @@ class Header {
   renderSettings() {
     let holder = document.createElement('DIV');
 
-    /** Add type selectors */
-    this.levels.forEach( level => {
-      let selectTypeButton = document.createElement('SPAN');
+    // do not add settings button, when only one level is configured
+    if (this.levels.length > 1) {
```
Comment (neSpecc) [inline src/index.js:117]: I would suggest to invert this condition:

```js
if (this.levels.length <= 1){
  return holder;
}

// ...
```

**Comment by natterstefan** [comment]
@neSpecc good catch, I've updated the PR.

File: src/index.js [hunk 2]
```diff
@@ -113,47 +113,53 @@ class Header {
...
+      * Highlight current level button
+      */
       if (this.currentLevel.number === level.number) {
         selectTypeButton.classList.add(this._CSS.settingsButtonActive);
       }
 
       /**
-       * Add SVG icon
-       */
+      * Add SVG icon
```
Comment (neSpecc) [inline src/index.js:135]: Something happened with jsdoc comments indentation.

Comment (natterstefan) [reply src/index.js:135]: Yeah, my prettier happend ;) Gonna change it.

**Comment by natterstefan** [comment]
@neSpecc I've updated the PR, can you take a look, please?

**Pull request merged**